ws-nextjs-app/pages/[service]/new_live/[id]/KeyPoints/index.tsx

```diff
@@ -0,0 +1,69 @@
+/** @jsx jsx */
+
+import { jsx, useTheme } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import LegacyText from '#app/legacy/containers/Text';
+import LegacyParagraph from '#app/legacy/containers/Paragraph';
+import BulletedList from '#app/legacy/containers/BulletedList';
+import Blocks from '#app/legacy/containers/Blocks';
+import styles from './styles';
+import { SummaryListWrapper, ComponentToRenderProps } from './types';
+
+const KeyPoints = ({
+  keyPointBlocks,
+}: {
+  keyPointBlocks: SummaryListWrapper[];
+}) => {
+  const {
+    palette: { GREY_10 },
+  } = useTheme();
+
+  const listItems = keyPointBlocks[0].model.blocks[0].model.blocks;
+  if (listItems.length === 0) return null;
```

Comment:
You could optional chain everything here to be super safe, just in case the top array has at least one item, but the other property access keys don't exist or the blocks array is empty:

```suggestion
  const listItems = keyPointBlocks?.[0]?.model?.blocks?.[0]?.model?.blocks;
  if (!listItems) return null;
```

Reply:
Good point - I have updated it to be `if (!listItems || listItems.length === 0) return null;`. Is that ok?

Reply:
Yeah looks good!